CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.17.0 Release
+- Release Date: March 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - If **DisableLegacyMfa** setting in ``config.json`` is set to ``true`` and [multi-factor authentication](https://docs.mattermost.com/deployment/auth.html) is enabled, ensure your users have upgraded to mobile app version 1.17 or later. See [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html) for more details.
```

Comment:
There is one more compatibility note for customers as a result of #2627 

https://community.mattermost.com/core/pl/u8g8dt4z1jbhij1c6xj7krnfdy

/cc @esethna

Reply:
@jasonblais I've believe the one you are talking about is right below this one

Reply:
I just added it